content_cafe.py

```diff
@@ -27,30 +27,32 @@
 )
 from core.util.summary import SummaryEvaluator
 
-class ContentCafeCoverageProvider(CoverageProvider):
-    def __init__(self, _db, api=None, uploader=None):
-        self._db = _db
-        self.input_identifier_types = [Identifier.ISBN]
-        output_source = DataSource.lookup(_db, DataSource.CONTENT_CAFE)
-        self.mirror = ContentCafeCoverImageMirror(self._db, uploader=uploader)
-        self.content_cafe = api or ContentCafeAPI(
-            self._db, self.mirror, uploader=uploader
-        )
 
-        super(ContentCafeCoverageProvider, self).__init__(
-            "Content Cafe Coverage Provider",
-            self.input_identifier_types, output_source,
-            batch_size=25)
+class ContentCafeCoverageProvider(IdentifierCoverageProvider):
+    SERVICE_NAME = "Content Cafe Coverage Provider"
```

Comment:
Since Content Cafe only provides metadata for individual titles and doesn't have any relationship to the collection that title is in, I made it an IdentifierCoverageProvider instead of a CatalogCoverageProvider